package.json

```diff
@@ -98,7 +101,10 @@
     "node": ">= 6.x.x"
   },
   "tonicExampleFilename": "docs/runkit/simple.js",
-  "typings": "./index.d.ts",
+  "types": "./index.d.ts",
```

Comment:
Is it a common change from `typings` to `types`?

Reply:
`tsd` say me to use "types" instead "typings"
Maybe it something related to this:
https://stackoverflow.com/questions/39261204/typings-vs-types-npm-scope

Reply:
Thanks.